charts/fluid/fluid/templates/csi/daemonset.yaml

```diff
@@ -14,6 +14,7 @@ spec:
       labels:
         app: csi-nodeplugin-fluid
     spec:
+      serviceAccount: fluid-csi-admin
       tolerations:
       - operator: Exists
       #priorityClassName: system-node-critical
```

Comment:
I  suggest the networkMode of csi plugin is configurable. By default, it's container network. But the hostnetwork is also workable.

Reply:
Done. Currently use container network by default. Host network with ClusterFirstWithHostNet dnsPolicy is another option.